test/functional/tests/data_integrity/test_data_integrity_long.py

```diff
@@ -96,17 +96,30 @@ def test_data_integrity_5d_dss(filesystems):
         fio_run.io_depth(128)
         fio_run.blocksize_range([(start_size, stop_size)])
         for core in cores:
-            fio_job = fio_run.add_job()
-            fio_job.directory(core.mount_point)
-            fio_job.size(core.size)
-        fio_run.run()
+            start = 512
+            file_section_size = Size(14, Unit.GibiByte).value   # 14 GiB per IO class
+            for _ in range(11):  # 11 IO classes for different sizes of files
```

Comment:
This `11` looks arbitrary. How about this?
```python
while start < Size(2, Unit.GibiByte).value:
```

Reply:
I agree